rbd-volume/entrypoint.sh

```diff
@@ -7,6 +7,12 @@ set -e
 : ${RBD_FS:=xfs}
 : ${RBD_TARGET:=/mnt/rbd}
 
+MOUNT_OPTIONS=''
+
+if [ "$RBD_OPTS" == "ro" ]; then
```

Comment:
don't you want to add more mount options than ro?
why don't you simply check with -z if the variable is populated?